From 099c1b9b07fbaa069cdc4ae7da53a820e757cf03 Mon Sep 17 00:00:00 2001
From: Olivier Fourdan <ofourdan@redhat.com>
Date: Fri, 22 Feb 2019 13:43:54 +0100
Subject: [PATCH] mouse: Move "locate-pointer" to "interface" schema

Moving the gsettings key "locate-pointer" from "org.gnome.desktop.
peripherals.mouse" to "org.gnome.desktop.interface" to reflect the
changes in gnome-settings-daemon and gsettings-desktop-schema".

https://gitlab.gnome.org/GNOME/gnome-tweaks/merge_requests/35
https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas/merge_requests/19
https://gitlab.gnome.org/GNOME/mutter/merge_requests/453
https://gitlab.gnome.org/GNOME/gnome-shell/issues/981
https://gitlab.gnome.org/GNOME/gnome-settings-daemon/merge_requests/86
---
 gtweak/tweaks/tweak_group_keymouse.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/gtweak/tweaks/tweak_group_keymouse.py b/gtweak/tweaks/tweak_group_keymouse.py
index 4933991..02da8bc 100644
--- a/gtweak/tweaks/tweak_group_keymouse.py
+++ b/gtweak/tweaks/tweak_group_keymouse.py
@@ -376,9 +376,8 @@ TWEAK_GROUPS = [
                                 "accel-profile",
                                 schema_filename="org.gnome.desktop.peripherals.gschema.xml"),
         GSettingsSwitchTweak(_("Pointer Location"),
-                             "org.gnome.settings-daemon.peripherals.mouse",
+                             "org.gnome.desktop.interface",
                              "locate-pointer",
-                              schema_filename="org.gnome.settings-daemon.peripherals.gschema.xml",
                               desc=_("Press the Ctrl key to highlight the pointer.")),
         GSettingsSwitchTweak(_("Middle Click Paste"),
                              "org.gnome.desktop.interface",
-- 
2.22.0

